src/resources/pandoc/datadir/_json.lua

```diff
@@ -72,7 +88,10 @@ local function encode_table(val, stack)
 
   stack[val] = true
 
-  local n = 0
+  if next(val) == nil then
+    return '[]'
```

Comment:
this is consistent with previous behavior but an empty table could also be encoded as `{}`

Reply:
The correspondence between JSON's `{}`,`[]`, `null` and Lua's `{}` and `nil` can never be done perfectly, so we should just maintain the previous behavior.